public/css/main.css

```diff
@@ -715,15 +715,23 @@ textarea {
     margin-bottom: 1.2rem;
 }
 
-#commit.new:after,#commit.wew:after {
+.new:after,.wew:after {
     content: "new";
     color: red;
-    font-size: 10pt;
+    font-size: 8pt;
     vertical-align: top;
     line-height: 5px;
     margin-left: 1px;
 }
-#commit.wew:after { content: "wew"; }
+.wew:after { 
+    content: "wew";
```

Comment:
Shouldn't this be "new" instead of "wew"

Reply:
This is some old thing that i put because at time the commit hash would show "unknown", and as such i would show "wew" on it instead of "new"
It can be removed

Reply:
Oh ok, no keep it if you want. I thought it was a typo